study.md

```diff
@@ -47,14 +47,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+The exclamation point signifies that the original data is being mutated by the method, as opposed to simply creating a new copy.
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+instance = {}
 ```
```

Comment:
not quite. take a look at creating an instance of a class again. We would use the .new method here.